src/styles/index.scss

```diff
@@ -1,3 +1,215 @@
+:root {
+  --accent-color: #00acdc;
+  --main-text-color: #616070;
+  --accent-text-color: #060b35;
+  --white: #fff;
+  --border-color: #f3f3f3;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+/* #region header */
+
+ul {
+  list-style: none;
+}
+
+.header {
+  box-sizing: content-box;
+  display: flex;
+  justify-content: space-between;
+  height: 40px;
+  align-items: center;
+  padding: 10px 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.logo {
+  display: block;
+  align-self: center;
+}
+
+.nav__list-items {
+  padding: 0;
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__list-item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__list-link {
+  position: relative;
+
+  display: inline-block;
+  padding: 12.5px 0;
+  align-content: center;
+
+  height: 40px;
+  text-transform: uppercase;
+  line-height: 14.6px;
+  text-decoration: none;
+  color: #000;
+  text-align: center;
+}
```

Comment:
The `align-content` property on line 60 does not apply to inline-level elements like `inline-block`, and it is not necessary here. You can safely remove it to avoid confusion.